lib/services/presets.go

```diff
@@ -202,6 +196,66 @@ func NewPresetAuditorRole() types.Role {
 	return role
 }
 
+// NewPresetReviewerRole returns a new pre-defined role for reviewer. The
+// reviewer will be able to review all access requests.
+func NewPresetReviewerRole() types.Role {
+	enterprise := modules.GetModules().BuildType() == modules.BuildEnterprise
+	role := &types.RoleV6{
+		Kind:    types.KindRole,
+		Version: types.V6,
+		Metadata: types.Metadata{
+			Name:        teleport.PresetReviewerRoleName,
+			Namespace:   apidefaults.Namespace,
+			Description: "Review access requests",
+		},
+		Spec: types.RoleSpecV6{
+			Options: types.RoleOptions{
+				CertificateFormat: constants.CertificateFormatStandard,
+				MaxSessionTTL:     types.NewDuration(apidefaults.MaxCertDuration),
+				PortForwarding:    types.NewBoolOption(true),
+				ForwardAgent:      types.NewBool(true),
+				BPF:               apidefaults.EnhancedEvents(),
+				RecordSession:     &types.RecordSession{Desktop: types.NewBoolOption(true)},
+			},
+			Allow: types.RoleConditions{
+				ReviewRequests: defaultAllowAccessReviewConditions(enterprise)[teleport.PresetReviewerRoleName],
+			},
+		},
+	}
+	role.SetLogins(types.Allow, []string{"no-login-" + uuid.New().String()})
```

Comment:
What's this for?

Reply:
I added this (basically coping off `auditor`) to ensure that the reviewer/requester role can't log into anything on its own. In other words, it's purely for reviewing/requesting and nothing else.

Reply:
Wouldn't omitting logins completely achieve the same result?

Reply:
@mdwn can we get rid of this?

Reply:
Okay, I've removed this and from the other new roles. I've left the auditor role intact, however.